src/main/java/nextstep/fp/StreamStudy.java

```diff
@@ -27,7 +28,13 @@ public static void printLongestWordTop100() throws IOException {
 				.get("src/main/resources/fp/war-and-peace.txt")), StandardCharsets.UTF_8);
 		List<String> words = Arrays.asList(contents.split("[\\P{L}]+"));
 
-		// TODO 이 부분에 구현한다.
+		words.stream()
+                .filter(word->word.length()>12)
```

Comment:
```suggestion
                .filter(word -> word.length() > 12)
```
람다식을 사용하실 때도 space 규칙을 지켜주세요!